crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -98,7 +100,10 @@ pub fn cli() -> clap::Command {
         .after_help("Run `spacetime help publish` for more detailed information.")
 }
 
-pub fn exec(args: &clap::ArgMatches) -> anyhow::Result<()> {
+pub fn exec(config: Config, args: &clap::ArgMatches) -> anyhow::Result<()> {
+    // Release the lockfile on the config, since we don't need it.
+    drop(config);
```

Comment:
nit on this and others: it sounds like matching with `_` might implicitly do this?

If not, might be nice to add a `config.release_lock()` or something convenience function, so this docstring can be in one place and the code can self-document a bit more.

Reply:
Moving into a `_` pattern does drop, but I prefer being more explicit because I think that fact is not obvious. It would be a valid choice for Rust to make it so that moving into `_` defers the drop until the end of scope, similar to moving into an unused binding like `_foo`.

I like the `release_lock` idea.

Reply:
yeah it does seem easy to accidentally change semantics by "trivially" changing code (e.g. literally the variable names)

Reply:
Added a `release_lock` method.